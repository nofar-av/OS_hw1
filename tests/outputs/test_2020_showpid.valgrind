==26849== Memcheck, a memory error detector
==26849== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26849== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26849== Command: /home/omer/OS_wet1/smash
==26849== Parent PID: 26847
==26849== 
==26849== 
==26849== FILE DESCRIPTORS: 5 open at exit.
==26849== Open file descriptor 4: my_pid
==26849==    at 0x54EEC3E: open (open64.c:47)
==26849==    by 0x110A4F: RedirectionCommand::execute() (Commands.cpp:417)
==26849==    by 0x11A242: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==26849==    by 0x118BCD: main (smash.cpp:42)
==26849== 
==26849== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_2020_showpid.valgrind
==26849==    <inherited from parent>
==26849== 
==26849== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_2020_showpid.err
==26849==    <inherited from parent>
==26849== 
==26849== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_2020_showpid.out
==26849==    <inherited from parent>
==26849== 
==26849== Open file descriptor 0:
==26849==    <inherited from parent>
==26849== 
==26849== 
==26849== HEAP SUMMARY:
==26849==     in use at exit: 0 bytes in 0 blocks
==26849==   total heap usage: 82 allocs, 82 frees, 84,171 bytes allocated
==26849== 
==26849== All heap blocks were freed -- no leaks are possible
==26849== 
==26849== For counts of detected and suppressed errors, rerun with: -v
==26849== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
