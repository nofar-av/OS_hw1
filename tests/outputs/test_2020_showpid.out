smash> smash pid is 5262
smash> smash pid is 5262
smash> smash> smash pid is 5262
smash> smash pid is 5262
smash> smash pid is 5262
smash> smash> 